docs/arcadia-users-group/20240730-arcadiathemer/20240730-arcadiathemer.Rmd

```diff
@@ -0,0 +1,143 @@
+2. You can use `RStudio` or a Jupyter notebook for creating plots. Install `RStudio` for Mac OSx [here] or Jupyter in a conda environment [following these instructions]()
+3. To use the custom fonts you need to download the `TTF` formatted font files and place in the `Users/YOURUSERNAME/Library/Fonts/` directory. You can also double click on the fonts to install them using FontBook. Check out the Arcadia Science Brand Assets page in Notion to find these. This should only need to be performed once even if the package is updated over time. **You must download the TTF formatted files to be compatible with `arcadiathemeR`.** If you want to download both the `TTF` and `OTF` formatted files to use both `arcadiathemeR` and the `arcadia-pycolor` python package that's fine - there will just be duplicate selections for each font in Illustrator. 
+4. Install `arcadiathemeR` with the `remotes` package:
+
+    ```
+    # install.packages("remotes")
+    remotes::install_github("Arcadia-Science/arcadiathemeR")
+    ```
+5. Load the package to use in your scripts with `library(arcadiathemeR)`. When you first load the package it will print a message about whether the custom fonts have been accessed and loaded correctly.
+
+## Layering onto an existing `ggplot2` plot
+
+There are two main functions you will use to layer onto an existing `ggplot2` plot to create plots adhering to the Arcadia style guide and access palettes. These are the `theme_arcadia()` and `scale` functions. The particular `scale` function differs on whether you use `color` or `fill` to access colors. Here is how you use these functions without changing the default arguments:
+
+```{r, example_plot}
+library(ggplot2)
+library(arcadiathemeR)
+
+ggplot(data=mtcars, aes(x=hp, y=mpg, color=as.factor(cyl))) +
+  geom_point() +
+  theme_arcadia() +
+  scale_color_arcadia()
+```
+
+If you want to change the color palette used, you can access this with the `palette_name` argument in the `scale` function: 
+
+```{r, palette_change}
+ggplot(data=mtcars, aes(x=hp, y=mpg, color=as.factor(cyl))) +
+  geom_point() +
+  theme_arcadia() +
+  scale_color_arcadia(palette_name = "primary")
+```
+
+We have specific font specifications for whether the represented data is categorical or numerical. In the next plot where the x-axis is different categories, you can see where the font isn't specified correctly:
+```{r, fonts}
+ggplot(data=diamonds, aes(x=cut, fill=cut)) +
+  geom_bar() +
+  theme_arcadia() +
+  scale_fill_arcadia()
+```
+
+To use the correct font type, you can specify what type of data you have on each axis by using the `x_axis_type` or `y_axis_type` arguments in the `theme_arcadia()` function: 
+```{r, fonts_correct}
+ggplot(data=diamonds, aes(x=cut, fill=cut)) +
+  geom_bar() +
+  theme_arcadia(x_axis_type = "categorical") +
+  scale_fill_arcadia()
+```
```

Comment:
Probably add a line break here (not sure what the correct syntax is for Rmd) so that the text starts on the next line, rather than next to the plot in the Markdown document.

```suggestion
```</br>
```

Reply:
fixed all instances of this, I just had to have an additional line